src/cljs/chronograph_web/events/create-organization.cljs

```diff
@@ -0,0 +1,41 @@
+(ns chronograph-web.events.create-organization
```

Comment:
It looks like these are repeated in the page specific namespace. Maybe this is dead code?

Reply:
The views are using events from `chronograph-web.pages.create-organization.events`. So I think this whole namespace can probably go away.

Reply:
Looks dead to me too. Will remove it.

Reply:
This was dead code. We removed it.